### DISCUSSION TITLE
Uh oh. Something is totally broken here…

### DISCUSSION THREAD

**Discussion by Graemezee1** [akeeba/panopticon#705] [Q&A (Community Support)]
I have a couple of sites which will not connect,   I get this error  **Uh oh. Something is totally broken here…**
I have followed  the instructions several times  on several occasions with no joy 
 
Error message 
Normally we can detect the nature of the connection error and provide some useful troubleshooting information here, or at least point you to some kind of useful, third-party troubleshooting resource. When this is not the case, we can at least print out an error message which will help us help you troubleshoot it.
Instead, we got an error which we should be able to provide troubleshooting information for (Akeeba\Panopticon\Exception\SiteConnection\APIApplicationIsBroken), but we don't have the code for it. This should have not happened.
Please delete all files and folders inside the tmp folder of Akeeba Panopticon. This addresses the problem of this page's cached copy being severely out of date. If this doesn't help, please let us know.

Extra information I am using this as a tracking and access site  I cannot get a server-based Cron job with a one-minute interval it is 5 minutes, I note that using a web-based  Cron job with the number of sites I have is inadvisable.
Using Panopticon without automation is a much simpler way to manage my sites than my previous manual spreadsheet-based system. I appreciate  that  this may be part of the problem.

**Comment by nikosdion** [comment]
Most likely a `system` or `webservices` plugin on your server modified the request we sent, resulting in an error. Please check your installed plugins.
